cli.js

```diff
@@ -82,15 +95,28 @@ var changelogOpts = {
   config: flags.config
 };
 
+var githubOpts = Object
+  .keys(flags)
+  .filter(function (key) {
+    return /github/.test(key)
+  })
+  .reduce(function (config, key) {
+    var prop = key.replace(/github([A-Z])/, function (m, char) {
+      return char.toLowerCase()
+    })
+    config[prop] = flags[key].toString()
+    return config
+  }, {
+    type: 'oauth',
+    token: flags.token || process.env.CONVENTIONAL_GITHUB_RELEASER_TOKEN
+  })
+
 if (flags.verbose) {
   changelogOpts.debug = console.info.bind(console);
   changelogOpts.warn = console.warn.bind(console);
 }
 
-conventionalGithubReleaser({
-  type: 'oauth',
-  token: flags.token || process.env.CONVENTIONAL_GITHUB_RELEASER_TOKEN
-}, changelogOpts, templateContext, function(err, data) {
+conventionalGithubReleaser(githubOpts, changelogOpts, templateContext, function(err, data) {
```

Comment:
You mean merge ``githubOpts`` and ``changelogOpts`` together?
I like less arguments, but it would became a breaking change (major release), is that ok?